public/index.html

```diff
@@ -0,0 +1,741 @@
+                <h2>Liz Camron</h2>
+                <p>Developer</p>
+              </li>
+            </ul>
+          </section>
+          <section class="f1-reviews-featured">
+            <div class="underlined-heading">
+              <h1 class="underlined-heading-text">AppBox Featured In ...</h1>
+              <div class="underlined-heading-underline"></div>
+            </div>
+            <div><img src="./img/Logos-gray.png" alt="The logs of all the entities that have featured AppBoX"></div>
+          </section>
+          <section class="f2-reviews-contact" id="support">
+            <svg class="slant" xmlns="http://www.w3.org/2000/svg" version="1.1" width="100%" height="100" viewBox="0 0 100 102" preserveAspectRatio="none">
+              <path d="M0 0 L0 100 L100 0 Z" />
+            </svg>
+            <div class="underlined-heading">
+              <h1 class="underlined-heading-text">App Support</h1>
+              <div class="underlined-heading-underline"></div>
+            </div>
+            <article class="contact-article">
+              <ul class="contact-info">
+                <li>
+                  <figure class="container-icon"><p class="icon-pin"></p></figure>
+                  <div class="container-icon-caption">
+                    <h1 class="container-icon-title">Address</h1>
+                    <p>121 King Street, Melbourne<br>Victoria, Australia 3000</p>
+                  </div>
+                </li>
+                <li>
+                  <figure class="container-icon"><p class="icon-phone"></p></figure>
+                  <div class="container-icon-caption">
+                    <h1 class="container-icon-title">Free Helpline</h1>
+                    <p>(001) 800 345 333700<br>(002) 600 345 333700</p>
+                  </div>
+                </li>
+                <li>
+                  <figure class="container-icon"><p class="icon-email"></p></figure>
+                  <div class="container-icon-caption">
+                    <h1 class="container-icon-title">Email</h1>
+                    <p>info@appbox.com<br>feedback@appbox.com</p>
+                  </div>
+                </li>
+              </ul>
+              <section class="form-section">
+                <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce.Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce.</p>
+                <form class="contact-form">
+                  <div class="contact-form-first-line">
+                    <input type="text" name="fullname" placeholder="Full Name">
+                    <input type="email" name="email" placeholder="Email">
+                  </div>
+                  <textarea rows="4" cols="50" placeholder="Message"></textarea>
+                </form>
+                <input class="contact-form-submit-button" type="submit" value="SEND">
+              </section>
+            </article>
+          </section>
+          <a href="home"></a>
```

Comment:
I _think_ this is supposed to be the arrow at the top of the footer that returns the user to the top of the page, but it doesn't get displayed at all (no content, no height, etc). Your `href` value also won't work (it needs the leading `#` character both for native behavior and for the smooth scrolling script that you've added). It's a little detail that is by contrast perhaps not all that important, but it's a huge UX improvement and certainly something that your designer is going to pick up on as being missed. In its current state, it's also a useless node as no user can actually interact with it.